app/(api)/_datalib/collections/createCollectionItem.ts

```diff
@@ -4,7 +4,7 @@ import { ObjectId } from 'mongodb';
 import { getDatabase } from '../../_utils/mongodb/mongoClient.mjs';
 import { HttpError, NoContentError } from '../../_utils/response/Errors';
 
```

Comment:
probably want to parse and replace any JSON input that comes from the body. Doesn't hurt to just do it.